pkg/controllers/clusteroperator/clusteroperator_test.go

```diff
@@ -0,0 +1,112 @@
+package clusteroperator
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/ibm-roks-toolkit/pkg/cmd/cpoperator"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+)
+
+var (
+	cfg          *rest.Config
+	k8sClient    kubeclient.Interface
+	configClient configclient.Interface
+	testEnv      *envtest.Environment
+	doneMgr      = make(chan struct{})
+	ctx          = context.Background()
+)
+
+func TestClusterOperator(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	RunSpecsWithDefaultAndCustomReporters(t,
+		"Control Plane Operator (Cluster Operator Controller) Suite",
+		[]Reporter{printer.NewlineReporter{}})
+}
+
+var _ = BeforeSuite(func() {
+	By("bootstrapping test environment")
+	testEnv = &envtest.Environment{
+		CRDDirectoryPaths: []string{filepath.Join("clusteroperator.crd.yaml")},
```

Comment:
looks like filepath.Join is not needed here

Reply:
Actually it may make sense to put the crd file in a separate directory to not mix code and test artifacts